doc/how_to_guides/usage.md

```diff
@@ -16,10 +16,12 @@ be achieved with either the Rails generator or the Rake task
 #### API
 ```
 Usage:
-  rails generate onesie:task NAME PRIORITY [options]
+  rails generate onesie:task NAME [options]
 
 Options:
   [--skip-namespace], [--no-skip-namespace]  # Skip namespace (affects only isolated applications)
+  [--priority] # Specify a priority that the onesie should run as
+  [--template] # Specify a custom template for generating the onesie
```

Comment:
@timlkelly let me know if this is what you'd expect. I figured since we now accept multiple args, using options is acceptable like this.